components/proxy/src/test/java/com/hotels/styx/admin/handlers/json/JsonReformatterTest.java

```diff
@@ -107,7 +107,8 @@ public void createsMetricsJsonCorrectly() throws JsonProcessingException {
                 "\n" +
                 "  },\n" +
                 "  \"version\":\"3.1.3\"\n" +
-                "}"));
+                "}";
+        assertThat(after, is(expectedString.replace("\n", System.lineSeparator())));
```

Comment:
We should try to find a consistent way to handle the line-separator cross-platform issue, so that we are not cluttering the code too much.